utils/ansible-freeipa.spec.in

```diff
@@ -62,7 +62,7 @@ cp -rp tests %{buildroot}%{_datadir}/ansible-freeipa/
 %{_datadir}/ansible/roles/ipaclient
 %{_datadir}/ansible/plugins/module_utils
 %{_datadir}/ansible/plugins/modules
-%doc README*.md
+%doc docs/modules/*.md
```

Comment:
The base README.md file also needs to be here.

Reply:
ok.